packages/astro/e2e/fixtures/dev-toolbar/src/pages/audits-perf.astro

```diff
@@ -0,0 +1,9 @@
+---
+import { Image } from "astro:assets";
+---
+
+<Image src="https://source.unsplash.com/random/800x600" loading="lazy" width={800} height={600} alt="Random Image" />
+
+<div style="height: 9000px;">
+
+<Image src="https://source.unsplash.com/random/800x600" loading="eager" width={800} height={600} alt="Random Image" />
```

Comment:
Is it safe to use unsplash in tests? Also, the div here isn't working as expected I think because you didn't close its tag, so the second image is inside the div and becomes in the fold.